src/Expression/Expression.php

```diff
@@ -24,7 +24,7 @@ public function __construct(Path $path, CacheItemPoolInterface $cache = null, $p
         $this->addDefaultFunctions();
     }
 
-    public function evaluate($expression, $values = [])
+    public function evaluate(\Symfony\Component\ExpressionLanguage\Expression|string $expression, array $values = []): mixed
```

Comment:
we should update the cs fixer to fix this to an import rather than an FQN, if it's not already failing on this :) but not important.

Reply:
hmmm I need to update the workflow anyway ... not seen Github workflows before...

Reply:
php-cs-fixer close to releasing with 8.1 support but not there yet. I will disable it for now in 0.3.x branch.